### ISSUE TITLE
Use the source distribution in the build instructions

### ISSUE THREAD

**Issue by AtnNn** [rethinkdb/docs#325]
The [build instructions](http://rethinkdb.com/docs/build/) and the installation page for each platform recommend doing a shallow clone:

```
git clone --depth 1 -b v1.12.x https://github.com/rethinkdb/rethinkdb.git
```

It is usually better for users to use the source distribution:

```
wget http://download.rethinkdb.com/dist/rethinkdb-latest.tgz
tar xf rethinkdb-latest.tgz
```

The tarball contains the v8 source and pre-compiled web assets. Users who just want to build rethinkdb can do so faster and with fewer dependencies. It also avoids many common npm and v8 related build failures.

Perhaps instructions for cloning the git repository and installing the extra dependencies can be moved to the "Hack on the database" section of the website.

**Comment by AtnNn** [comment]
From #619:

> Specifically mentioning
> > They contain pre-compiled platform-independent assets and pre-fetched dependencies.
> > As that's excessively relevant.

**Comment by AtnNn** [comment]
Fixed in 6827dd6
